src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/Avx512F.cs

```diff
@@ -1833,22 +1895,52 @@ public new abstract class X64 : Avx2.X64
         /// </summary>
         public static Vector128<float> ConvertScalarToVector128Single(Vector128<float> upper, uint value) => ConvertScalarToVector128Single(upper, value);
         /// <summary>
+        /// __m128 _mm_cvt_roundi32_ss (__m128 a, int b, int rounding)
+        /// VCVTSI2SS xmm1, xmm2, r32 {er}
+        /// </summary>
+        public static Vector128<float> ConvertScalarToVector128Single(Vector128<float> upper, int value, [ConstantExpected(Max = FloatRoundingMode.ToZero)] FloatRoundingMode mode) => ConvertScalarToVector128Single(upper, value, mode);
```

Comment:
Looks like we're missing the unsigned equivalent: `ConvertScalarToVector128Single` (`VCVTUSI2SS`)